src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+import {
+  Pagination as CorePagination,
+  PaginationProps as CorePaginationProps,
+} from 'wix-ui-core/pagination';
+
+import { DoubleChevronLeft } from '../../assets/icons/DoubleChevronLeft';
+import { DoubleChevronRight } from '../../assets/icons/DoubleChevronRight';
+import { ChevronLeft } from '../../assets/icons/ChevronLeft';
+import { ChevronRight } from '../../assets/icons/ChevronRight';
+
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+
+import styles from './Pagination.st.css';
+
+export interface PaginationProps extends CorePaginationProps {}
+
+/** Pagination component */
+export const Pagination: React.FunctionComponent<PaginationProps> = props => {
+  return (
+    <TPAComponentsConsumer>
+      {({ mobile: mobileView, rtl }) => {
+        return (
+          <CorePagination
+            {...styles('root', { mobileView, rtl }, props)}
+            previousLabel={rtl ? ChevronLeft : ChevronRight}
+            nextLabel={!rtl ? ChevronLeft : ChevronRight}
```

Comment:
I think this is flipped, no?

Reply:
Oh, btw, I wasn't aware of the `rtl` prop that the `core` component has.
I think that it means we don't need to add anything special in our code, because I think it automatically flips the prev/next buttons.

Reply:
yup. it's swaps prev/next buttons - so icons looks weird (i.e. prev button with next icon etc)

Reply:
So maybe this `rtl ? ChevronLeft : ChevronRight` isn't needed?

Reply:
w/o this fix:
<img width="554" alt="Screen Shot 2019-06-26 at 6 14 21 PM" src="https://user-images.githubusercontent.com/846659/60192328-57350e80-983e-11e9-8421-59dfa6d05787.png">
with:
<img width="554" alt="Screen Shot 2019-06-26 at 6 14 41 PM" src="https://user-images.githubusercontent.com/846659/60192331-57cda500-983e-11e9-958c-64fc5e65b97f.png">

Reply:
you are right)
rechecked - corrected